querydsl-jpa/pom.xml

```diff
@@ -48,30 +48,13 @@
         </exclusion>
       </exclusions>      
       <optional>true</optional>
-    </dependency>
-    <dependency>
-      <groupId>org.hibernate</groupId>
-      <artifactId>hibernate-entitymanager</artifactId>
```

Comment:
`hibernate-entitymanager`  no longer exists, as it has been integrated into  `hibernate-core`.